dpkg/config.yml

```diff
@@ -0,0 +1,54 @@
+log-type: Syslog
+keys-file: /var/lib/tox-node/keys
+udp-address: 0.0.0.0:33445
+tcp-addresses:
+  - 0.0.0.0:33445
+tcp-connections-limit: 16383 # 16383 tcp + 1 for udp = 16384
```

Comment:
Nope, 16383 is not enough. We allow one more TCP connection than `tcp-connections-limit` specifies since we can check the limit only after TCP connection was accepted. Also, I'm not sure that even 16382 is enough because mio creates auxiliary pipes which might be counted as well. You should either test it or specify significantly lower value (like twice lower).